lib/Macros/Sources/SwiftMacros/SwiftifyImportMacro.swift

```diff
@@ -468,7 +468,14 @@ struct FunctionCallBuilder: BoundsCheckedThunkBuilder {
     let functionRef = DeclReferenceExprSyntax(baseName: base.name)
     let args: [ExprSyntax] = base.signature.parameterClause.parameters.enumerated()
       .map { (i: Int, param: FunctionParameterSyntax) in
-        return pointerArgs[i] ?? ExprSyntax("\(param.name)")
+        if let overrideArg = pointerArgs[i] {
```

Comment:
I wonder if this is a fix we want to backport to 6.2.

Reply:
Yeah that's probably a good idea